### ISSUE TITLE
Staging Review finding: Process list step contains heading with no content

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#98740]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Disability Benefits Experience Team 1
**Product:** Disability Comp Application (21-526 EZ)
**Feature:** Submission Experience

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that does not meet best practices.](https://design.va.gov/about/experience-standards/#user-encounters-content-that-meets-best-practices)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On the confirmation page, the process list under "What to expect" has a first step of "We'll send you an email to confirm your submission," which is coded as an H3.

Per the [component guidance](https://design.va.gov/components/process-list#accessibility-considerations), process list steps shouldn't be a heading alone. Instead, there should be some meaningful content that follows the heading describing that step in more detail.

When a screen reader user encounters a heading, the expectation is that the heading describes the content that follows it. When there's no content after it, or the next thing is a heading describing a section at an equal level, there may be some confusion --- did some content not load? Is there more I should be looking for? etc.

<details><summary>Link, screenshot or steps to recreate</summary>

![What to expect process list with two steps. The first step has eyebrow text "Active" and a heading that reads "We'll send oyu an email to confirm your submission." The second step has eyebrow text "Pending" and a heading that reads "Next we'll send you a letter to let you know we have your claim" followed by a paragraph reading "You should get this letter in about 1 week, plus mailing time, after we receive your claim."](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/86117-Brian-1.png?raw=true)

</details>

## Recommended action

Include some additional description of the email step. Even a perfunctory description will help avoid that small amount of friction for screen reader users.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jeana-adhoc** [comment]
The pattern allows teams to define the additional content needed in the review page. This is not currently being defined in the teams code. There's nothing here for the design system to fix.

[See this line of code](https://github.com/department-of-veterans-affairs/vets-website/blob/23efe0e0a2a056b13d441a67d51b898266f0be7b/src/applications/disability-benefits/all-claims/containers/ConfirmationPage.jsx#L65)

The documentation example gives examples for what this component should say. It has also been updated and no longer utilizes eyebrow text.

[See the pattern in VADS](https://design.va.gov/patterns/help-users-to/keep-a-record-of-submitted-information)

**Comment by amylago** [comment]
C&IA shared [their content recommendation](https://www.figma.com/design/zduybBnXDjVkKpJosfC6oh/526---Submission---Provide-a-Copy-Update-7.2?node-id=2480-5715&t=b69Vsm0wswmK6svT-4). Also shared in the [content ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/135917#issuecomment-4270132242).
